### PR TITLE
Member one done

### PR DESCRIPTION
**Pull request by DyaryRaoof**
- Create a Book class in a separate .rb file.
- Create a Label class with an association to the Item class (in a separate .rb file).
- All Book class properties visible in the diagram should be defined and set up in the constructor method.
- All Label class properties visible in the diagram should be defined and set up in the constructor method.
- Implement methods:
- add_item method in the Label class
  - should take an instance of the Item class as an input
  - should add the input item to the collection of items
  - should add self as a property of the item object (by using the correct setter from the item object)
- can_be_archived?() in the Book class
  - should override the method from the parent class
  - should return true if parent's method returns true OR if cover_state equals to "bad"
  - otherwise, it should return false
- Add unit tests for all implemented methods.
- The following options should be available:
  - List all books
  - List all labels (e.g. 'Gift', 'New')
  - Add a book
- All data should be preserved by saving collections in .json files.
- Create a schema.sql file with tables that will be analogical to the structure of the classes that you created:
  - books table (add all properties and associations from the parent Item class as table columns)
  - labels table

### CODE REVIEW

File: books_manager.rb [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  def add_book
+    puts 'Enter publisher'
+    publisher = gets.chomp
+    puts 'Enter cover state'
+    cover_state = gets.chomp
+    puts 'Enter publish date'
+    publish_date = gets.chomp
+    puts 'Enter true or false if the books is archived '
+    archived = gets.chomp
+    book = Book.new(publisher, cover_state, publish_date, archived: archived)
```
Comment (lfmnovaes) [inline books_manager.rb]: Consider creating the object without the `archived` variable, then use the `move_to_archive` method in the class.
Also, you should convert Y/N from user input to true/false for the object

File: labels_manager.rb [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+    end
+  end
+
+  private
+
+  def convert_hashs_to_labels(hashes)
+    labels = []
+    hashes.each do |h|
+      l = Label.new(h['title'], h['color'], h['id'])
+      # l.items.each do |_item_hash|
```
Comment (lfmnovaes) [inline labels_manager.rb]: Remove all unnecessary comments from your project

**Review by lfmnovaes — CHANGES_REQUESTED** [review]
### Excellent work @DyaryRaoof 💪
There are some minor issues that need to be fixed. Kindly work on them.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

File: labels_manager.rb (hunk 2 above)
Comment (DyaryRaoof) [reply labels_manager.rb]: but shouldn't the archived variable be there already? what if the object is in a bad state and older than 10 years so it should be archived already

**Review by lfmnovaes — APPROVED** [review]
### Nice job @DyaryRaoof 👍 

Great work on implementing the required changes.
There is nothing else to say other than **it's time to merge**.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**